src/classes/Callable_API.cls

```diff
@@ -95,6 +95,10 @@ global with sharing class Callable_API implements System.Callable {
                 // parameter service depending on which action was invoked
                 return CallableDispatchService_ERR.getInstanceOf(paramService).dispatch();
 
+            } when 'settings.enableenhancedrecurringdonations' {
```

Comment:
@force2b , can the API request be called `RD2.EnableEnhancement`? The current one is a bit too long and it is indicates the settings, but let's put all RD2 checks into the RD2 context as we did for the CRLP methods?

Reply:
What about either `Settings.EnableEnhancedRecurringDonations` or `RD2.Enable`? I feel like the former is more consistent with `Settings.EnableCustomizableRollups` and what will be `Settings.EnableAdvancedBDIMapping`.

Reply:
Oh I see, there is `Settings.EnableCustomizableRollups` in another PR and a new WI for `Settings.EnableAdvancedBDIMapping`. Ok, let's leave it as is, ie `Settings.EnableEnhancedRecurringDonations`.